### ISSUE TITLE
Screen reader reads "link" for status entries

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#99042]
Testing #97156

I noticed that going through focused entries of the status bar, they are read as "link...". I think they should be read as buttons?

**Comment by isidorn** [comment]
Good catch and makes sense. We will change the role of items in the statusbar to `button`.
fyi @marcozehe @jvesouza

**Comment by MarcoZehe** [comment]
Sounds very good and is in line with Word and other such bars that have buttons, be it tool or status. ;-)
